src/chi/utilities/sizing.scss

```diff
@@ -0,0 +1,325 @@
+@import 'variables';
+@import 'mixins';
+
+.chi {
+
+  //Width Auto
+  .-w--auto {
+    width: auto;
+  }
+
+  .-w-sm--auto {
```

Comment:
@dani-cl-madrid @jllr would you mind walking David through how to simplify this with a loop or mixin similar to our other utilities?

Reply:
@mattnickles sure